packages/nimble-components/src/rich-text/specs/README.md

```diff
@@ -107,6 +111,159 @@ Example usage of the `nimble-rich-text-editor` in the application layer is as fo
 </nimble-rich-text-editor>
 ```
 
+Additionally, this rich text editor also includes a feature for tagging or mentioning a user by inserting the **"@"** symbol within the editor.
```

Comment:
This is a very large feature addition. We usually have separate documents for that kind of addition, for example see how the table places features like sorting and grouping in separate documents and links back and forth from the top-level table HLD: https://github.com/ni/nimble/tree/1d72f6c1118c17e0dd578697d3acbfd11363831d/packages/nimble-components/src/table/specs#api

Things that impact the top-level <rich-text-viewer> or <rich-text-editor> elements can stay here such as the event names that are fired and link to the correct section in the feature doc.

We had originally discussed not needing to split the document with @m-akinc  and @atmgrifter00  but in trying to review this it would be a lot easier to see the changes in the API surface id the docs were split.

Reply:
Sure, I have split this into a separate HLD for the `mention` feature but not for each sub-component that we are planning to create. I just cross-linked from the actual readme file. linking to relevant sections to branch out the mention support into a separate file. I hope this level of branching out is sufficient for now and makes accessing the mention support content straightforward.